### PR TITLE
Julia-bindings proposal

### PR DESCRIPTION
**Pull request by peremato**

### CODE REVIEW

File: _gsocorgs/2023/cea-irfu.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "CEA-IRFU"
+author: "Pere Mato"
+layout: default
+organization: 
+logo: CEA-logo.png
+description: |
+The [IRFU](https://irfu.cea.fr), Institute for Research on the fundamental laws of the universe, of the Fundamental Research Division of the [CEA](https://www.cea.fr), brings together three scientific disciplines, astrophysics, nuclear physics and particle physics, as well as al the associated technological expertise.
```
Comment (vvolkl) [inline _gsocorgs/2023/cea-irfu.md:8]:
```suggestion
  The [IRFU](https://irfu.cea.fr), Institute for Research on the fundamental laws of the universe, of the Fundamental Research Division of the [CEA](https://www.cea.fr), brings together three scientific disciplines, astrophysics, nuclear physics and particle physics, as well as al the associated technological expertise.
```

File: _gsocprojects/2023/project_HSF.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+project: HSF
+title: HSF
+layout: default
+logo: hsf_logo_angled.png
+description: |
+  The HEP Software Foundation encourages cooperation and common
+  development of software in High Energy Physics. The HSF hosts
+  a number of software projects where developers are working to
+  solve current problems in particle physics.
```
Comment (vvolkl) [inline _gsocprojects/2023/project_HSF.md:10]:
```suggestion
    The HEP Software Foundation encourages cooperation and common development of software in High Energy Physics. The HSF hosts a number of software projects where developers are working to solve current problems in particle physics.
```

**Review by vvolkl — CHANGES_REQUESTED** [review]
Thanks Pere - just some minor fixes to make sure jekyll renders this correctly.

I think you'll also need to rename `_gsocproposals/2023/proposal_Julia-HEP-C++.md` to `_gsocproposals/2023/proposal_Julia-HEP-Cpp.md` for the file to be recognised.

**Comment by peremato** [comment]
@vvolkl if you are happy with it, can be merged.

**Review by hegner — APPROVED** [review]

**Comment by hegner** [comment]
I did a minor fix myself but otherwise fine. Thanks!

**Pull request merged**